package.lisp

```diff
@@ -21,6 +23,13 @@ All ARGS are declared as `ignorable'."
                                              lambda-list-keywords)))
        ,@body)))
 
+(defun slot-names (class-specifier)
+  ;; TODO: `slot-names' or `direct-slot-names'?
```

Comment:
I think that `slot-names` makes more sense since it includes slots in superclasses, which is exactly the way you did it :)